core/cont/inc/TObjArray.h

```diff
@@ -56,35 +56,35 @@ friend class TClonesArray;
    TObjArray(const TObjArray &a);
    virtual          ~TObjArray();
    TObjArray& operator=(const TObjArray&);
-   virtual void     Clear(Option_t *option="");
+   virtual void     Clear(Option_t *option="") override;
    virtual void     Compress();
-   virtual void     Delete(Option_t *option="");
+   virtual void     Delete(Option_t *option="") override;
    virtual void     Expand(Int_t newSize);   // expand or shrink an array
-   Int_t            GetEntries() const;
+   Int_t            GetEntries() const override;
    Int_t            GetEntriesFast() const {
       return GetAbsLast() + 1;   //only OK when no gaps
    }
    Int_t            GetEntriesUnsafe() const;
-   Int_t            GetLast() const;
+   Int_t            GetLast() const override;
    TObject        **GetObjectRef() const { return fCont; };
-   TObject        **GetObjectRef(const TObject *obj) const;
-   Bool_t           IsEmpty() const { return GetAbsLast() == -1; }
-   TIterator       *MakeIterator(Bool_t dir = kIterForward) const;
-
-   void             Add(TObject *obj) { AddLast(obj); }
-   virtual void     AddFirst(TObject *obj);
-   virtual void     AddLast(TObject *obj);
-   virtual void     AddAt(TObject *obj, Int_t idx);
+   TObject        **GetObjectRef(const TObject *obj) const override;
+   Bool_t           IsEmpty() const override { return GetAbsLast() == -1; }
+   TIterator       *MakeIterator(Bool_t dir = kIterForward) const override;
+
+   void             Add(TObject *obj) override { AddLast(obj); }
+   virtual void     AddFirst(TObject *obj) override;
```

Comment:
Can you remove `virtual` for cases where you added `override`? The latter is enough.